src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/InterfaceDispatchCellNode.cs

```diff
@@ -69,7 +120,7 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
                 }
                 else
                 {
-                    throw new NotImplementedException();
+                    objData.EmitInt(interfaceMethodSlot);
```

Comment:
The entire `if` check can be replaced with `objData.EmitNaturalInt(interfaceMethodSlot)`